src/System.Collections.Specialized/src/Resources/Strings.resx

```diff
@@ -153,16 +153,4 @@
   <data name="OrderedDictionary_ReadOnly" xml:space="preserve">
     <value>The OrderedDictionary is read-only and cannot be modified.</value>
   </data>
-  <data name="ArgumentNull_Array" xml:space="preserve">
-    <value>Array cannot be null.</value>
-  </data>
-  <data name="Arg_ArrayPlusOffTooSmall" xml:space="preserve">
-    <value>Destination array is not long enough to copy all the items in the collection. Check array index and length.</value>
-  </data>
-  <data name="NotSupported_KeyCollectionSet" xml:space="preserve">
-    <value>Mutating a key collection derived from a dictionary is not allowed.</value>
-  </data>
-  <data name="NotSupported_ValueCollectionSet" xml:space="preserve">
-    <value>Mutating a value collection derived from a dictionary is not allowed.</value>
-  </data>
```

Comment:
Can you separate the product changes out into a separate commit?  This is about removing dead code, right?